src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.tsx

```diff
@@ -1,19 +1,19 @@
-import PropTypes from 'prop-types';
```

Comment:
You can ignore the diffs in the file. I just copied `installedIntegrationInDirectory.tsx` into here with the only modification being to remove the TODO item.